packages/paste-website/src/pages/form/input/index.mdx

```diff
@@ -0,0 +1,578 @@
+
+## When to use an input
+
+Use an input when users are expected to enter less than a single line of text, or text that’s shorter than a sentence.
+
+<DoDont>
+  <Do title="Do" body="Use an input to encourage short text entry."  preview={false}/>
+  <Dont
+    title="Don't"
+    body="Don’t use an input when text entry is expected to be longer than the width of the input since users may need to review the text before submission. Use a textarea instead."
+    preview={false}
+  />
+</DoDont>
+<DoDont>
+  <Do
+    title="Do"
+    body="If you limit the length of text entry, show a character counter and explain to users in help text why their entry is restricted."
+    preview={false}
+  />
+  <Dont
+    title="Don't"
+    body="Don’t have a character limit if you can’t explain to the user why their text entry is restricted."
+    preview={false}
+  />
+</DoDont>
+<DoDont>
+  <Do title="Do" body="Use help text to help users prevent errors and fill out a form field correctly."  preview={false}/>
+  <Dont title="Don't" body="Don’t use placeholder text for validation instructions."  preview={false}/>
+</DoDont>
+<DoDont>
+  <Do
+    title="Do"
+    body="Use the correct input type to help users format their text correctly and bring up dynamic keypads."
+    preview={false}
+  />
+  <Dont
+    title="Don't"
+    body="Don’t use the default input type if there’s a more specific one that can help users enter valid text more efficiently."
+    preview={false}
+  />
+</DoDont>
+<DoDont>
+  <Do
+    title="Do"
+    body="Keep help text and error text concise and simple. If you need to use more than 2 sentences to explain a field, link out to supporting docs or trigger a popover instead."
+    preview={false}
+  />
+  <Dont title="Don't" body="Don’t use more than 2 sentences in help text or error text." preview={false}/>
+</DoDont>
+<DoDont>
+  <Do title="Do" body="Include a visible label on every form field."  preview={false} />
+  <Dont title="Don't" body="Don’t use prefix/suffix text as a replacement for a label." preview={false} />
+</DoDont>
+<DoDont>
+  <Do
+    title="Do"
+    body="Use a disabled form field to show users that they can’t interact with the field, but that it could potentially be enabled through another UI element."
+    preview={false}
+  />
+  <Dont
+    title="Don't"
+    body="Don’t use a disabled form field to show information that can’t be edited. Instead, use a read-only form field or consider another way of showing static information."
+    preview={false}
+  />
+</DoDont>
+
+## Anatomy
```

Comment:
These anatomy tables aren’t scaling very well. We might need to rethink how we display these information in the future. It’s really hard to get a good mobile experience for them